open-craft/openedx-blockstore-relay
# Bumps version to 0.1.1

**Description:** Bumps version to allow us to tag and deploy the changes included in the past few PRs, specifically https://github.com/open-craft/openedx-blockstore-relay/pull/4 and https://github.com/open-craft/openedx-blockstore-relay/pull/5.

**JIRA:** LX-53

**Testing instructions:**

In a virtualenv, run:
```
pip install git+https://github.com/open-craft/openedx-blockstore-relay.git@35e0ea4#egg=openedx-blockstore-relay==0.1.1
```
and ensure that the installed version is `0.1.1`.

**Reviewers:**

**Merge checklist:**
- [x] CI build is green (ran manually)
- [ ] <strike>Documentation updated (not only docstrings)</strike> N/A

**Post merge:**
- [ ] <strike>Check new version is pushed to PyPi after tag-triggered build is finished.</strike> N/A
